language/nl.json

```diff
@@ -57,11 +57,13 @@
         },
         {
           "label": "Vervangende kleur voor transparant",
+          "description": "Stel de kleur in die de transparante gebieden van de afbeeldingen vervangt.",
+          "default": "#000000"
```

Comment:
This won't work :-)

Reply:
@viragom I know, it's only a missing comma, but I shouldn't start fixing pull requests myself if I don't want to open Pandora's box.